lib/console.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Console do
+  def print_board(board) do
+    IO.write """
+
+    --------------
+
+      #{Enum.at(board,0)}   #{Enum.at(board, 1)}   #{Enum.at(board, 2)}
+     ===+===+=== 
+      #{Enum.at(board, 3)}   #{Enum.at(board, 4)}   #{Enum.at(board, 5)}
+     ===+===+=== 
+      #{Enum.at(board, 6)}   #{Enum.at(board, 7)}   #{Enum.at(board, 8)}
+
+    --------------
+    
+    """
+  end
+
+  def print_welcome_msg do 
+    IO.write"""
+    Tic Tac Toe
+    -----------
+    Instructions:
+    Select the spot you want to mark by entering the corresponding number.
+    The first to get three marks in a row will win the game.
+
+    You are player "X", please start the game.
+    """
```

Comment:
At the moment, the console's job is to print messages. I'm not sure the console needs to know a lot about the specific messages it is printing. 

For example, if you wanted to change the language of your game, you would have to change the entire Console module. I would recommend having one `print` function, and then extracting the messages themselves to a separate file. For example: `Console.print(Messages.welcome)` or something like this.